sendto_silhouette.py

```diff
@@ -306,6 +306,17 @@ def recursivelyTraverseSvg(self, aNodeList,
         for node in aNodeList:
             # Ignore invisible nodes
             if isinstance(node, BaseElement):
+                # Check if layer name is referring to registration mark or print layer
```

Comment:
the whole check would only be needed on `Group`s, right? if so, move to Group logic below... this will help to speed things up